### PR TITLE
[#4418][#4471][#4472] New CI, Stdout, and Test mode (master)

### PR DESCRIPTION
**Pull request by korydraughn**
Passed in new CI.

### CODE REVIEW

File: scripts/irods/execute.py [hunk 1]
```diff
@@ -84,16 +84,25 @@ def execute_command_timeout(args, timeout=10, **kwargs):
             'The call {0} did not complete within'
             ' {1} seconds.'.format(args, timeout))
 
-def execute_command_permissive(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, input=None, **kwargs):
+def execute_command_permissive(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, input=None, foreground=None, **kwargs):
```
Comment (trel) [inline scripts/irods/execute.py]: foreground=False ?     rather than None?

File: scripts/irods/start_options.py [hunk 2]
```diff
@@ -27,3 +27,12 @@ def add_options(parser):
...
                       'timeout (ten minutes)')
+
+    parser.add_option('--stdout',
+                      dest='write_to_stdout', action='store_true',
+                      help='Write log messages to stdout instead of rsyslog')
+
+    parser.add_option('--test',
+                      dest='test_mode', action='store_true',
+                      help='Simultaneously writes log messages to stdout or rsyslog '
+                           'and IRODS_HOME/log/test_mode_output.log')
```
Comment (trel) [inline scripts/irods/start_options.py]: consider rephrasing to...

'Additionally write log messages to IRODS_HOME/log/test_mode_output.log'

File: scripts/irods/execute.py (hunk 1 above)
Comment (korydraughn) [reply scripts/irods/execute.py]: Thought I used `False` for that. Will change.

File: scripts/irods/start_options.py (hunk 2 above)
Comment (korydraughn) [reply scripts/irods/start_options.py]: Will change.

File: scripts/irods/execute.py (hunk 1 above)
Comment (korydraughn) [reply scripts/irods/execute.py]: Updated.

File: scripts/irods/start_options.py (hunk 2 above)
Comment (korydraughn) [reply scripts/irods/start_options.py]: Updated.

**Comment by korydraughn** [comment]
Added #.

**Pull request merged**